docs/runtimes/node-affinity.md

```diff
@@ -1,96 +1,214 @@
-# Node affinity for MLRun jobs
-Node affinity can be applied to MLRun to determine on which nodes 
-they can be placed. The rules are defined using custom labels on nodes and label selectors. 
-Node affinity allows towards Spot or On Demand groups of nodes.
+# Managing job resources
+
+Configuration of job resources is relevant for all supported cloud platforms.
+
+<!--
+  ## Function!!!!!! Resources: limit and request CPU, GPU, and Memory  
+
+  You can configure how much of each resource a function needs. Kubernetes uses this information when placing a pod on a node. The memory and 
+  CPU configurations that you specify in the service are applied to each replica. Limits and requests are supported for all services.
+
+  Service limit!
+
+  When creating a new function, set the **Memory** and **CPU** in the **Common Parameters** tab, under **Resources** .
+
+  Modify the Memory, CPU, GPU for an ML function by pressing **ML functions**, then press **<img src="../_static/images/kebab-menu.png" width="25"/>** 
+  of the function, and select **Edit** and scroll to the **Resources** section.
+-->
 
 **In this section**
-- [On demand vs spot](#On-demand-vs-spot)
-- [Stateless and stateful applications](#Stateless-and-stateful-applications)
-- [Node selector](#node-selector)
+- [Preemption mode: Spot vs. On-demand nodes](#preemption-mode-spot-vs-on-demand-nodes)
+- [Pod priority](#pod-priority)
+- [CPU, GPU, and memory limits for user jobs](#cpu-gpu-and-memory-limits-for-user-jobs)
```

Comment:
CPU/GPU/.. should be the first section (most widely used), Volumes should be the 2nd, followed by the rest